### DISCUSSION TITLE
Access Form variables from Blade?

### DISCUSSION THREAD

**Discussion by jacksleight** [statamic/cms#5533] [Q&A]
With the Blade changes in 3.3 is it possible to access the same Form variables that are available when using Antlers?

If I call `Statamic::tag('form:contact')->fetch()` I get an HTML string with the opening and closing `<form>` tags. Whereas in Antlers you have access to all the variables like `success`/`errors` and the `fields` array etc.

Is there another method I should be calling?

**Comment by jasonvarga** [comment]
We added the `Statamic::tag()` helper so it'll run the tag, but didn't actually try using each one of them. 

Looks like we'll need to tweak some tags to return stuff suitable for Blade. The form tags does some template parsing of its own, which obviously wouldn't apply to Blade. You can open an issue instead if you like.

**Reply by jacksleight** [reply]
Ah OK great, will submit an issue.

Before 3.3 I had a few helpers to deal with some of the quirks, but now Blade support is more official I'm hoping  I can ditch them!  😊

I'll check a couple of other things that I'm aware of as well.

**Reply by jasonvarga** [reply]
I think it will mostly be tags that actually alter the output between tag pairs in Antlers.

e.g. The `form` tag would wrap your tag pair contents with `<form>` and inject hidden inputs. In Blade that doesn't really translate since you're just kinda building out PHP.
